### PR TITLE
YALB-476: Wire Image Paragraph

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
### Description of work
- Wires the Image paragraph type

### Functional testing steps:
- [ ] Log in to the site https://pr-20-yalesites-platform.pantheonsite.io/user/login
- [ ] [Create a page](https://pr-20-yalesites-platform.pantheonsite.io/node/add). Fill in the title and add an "Image" component (with caption).
- [ ] Save the page, and verify the image and caption render on the page as expected ("content-width" wrapped in a `<figure>` tag)
- [ ] Edit the page, and remove the caption
- [ ] Save the page, and verify the image is just an `` tag now, without the figure markup. But that it's still "content-width".

### CODE REVIEW

File: scripts/local/setup.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Ignore the composer.lock file on local dev only.
 if grep -qxF 'composer.lock' .git/info/exclude; then
-  echo "Excluding composer.log to keep it out of the repo."
+  echo "Excluding composer.lock to keep it out of the repo."
```
Comment (nJim) [inline scripts/local/setup.sh:5]: Thanks 👍

**Review by nJim — APPROVED** [review]
Completes code review. Yaml files are easy. Everything looks great.

**Comment by ahitson** [comment]
Note to add cropping ticket.

**Review by ahitson — APPROVED** [review]

**Review by miketullo95 — APPROVED** [review]
This works great! 

I had one small hiccup with the last two image files in the test environment where they would not show up after saving. They are blank in the grid view as shown in SS but show up in the table view... odd 🤷 
Screen Shot 2022-05-03 at 12 58 19 PM

**Review by kara-franco — CHANGES_REQUESTED** [review]
Very nice! The only item I saw that needs a tweak is:

- [ ] the aria-label on the <figure> has some extra content coming in. In the example below, the aria-label should read "aria label is working"

Screen Shot 2022-05-03 at 1 32 14 PM

**Review by kara-franco — APPROVED** [review]

**Pull request merged**